nevergrad/functions/gymanm/test_gym_anm.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+from nevergrad.common import testing
+from . import gym_anm
+
+
+def test_gym_anm() -> None:
+    with testing.skip_error_on_systems(OSError, systems=("Windows",)):
```

Comment:
what was the problem on windows?

Reply:
It tells me than gym-anm is not present. No idea how this can happen on Windows and not Linux.

Reply:
Actually this fails at the import. Need to disable entirely this on Windows seemingly.